opendatahub/quickstart/README.md

```diff
@@ -1,28 +1,30 @@
-# Quick Start 
+# Overview of the OpenDataHub's ModelServing Quick Starts
 
-This documentation aims to assist you in utilizing OpenDataHub's ModelServing effectively. It focuses on explaining new features introduced in the OpenDataHub modelmesh or odh-model-controller and provides relevant examples to enhance comprehension.
+The purpose of these quick starts is help you learn how to use OpenDataHub's ModelServing. They describe features in the OpenDataHub ModelMesh or odh-model-controller and provide relevant examples.
 
-Please note that while these documents have been verified for accuracy at the time of their creation, there is a possibility that manifests or scripts may become outdated and not function as intended over time.
+**Note:** These quick starts have been verified for accuracy at the time of their creation, but the manifests or scripts might become outdated and not function as originally intended.
 
-Our objective is to provide comprehensive and professional guidance to ensure seamless utilization of OpenDataHub's ModelServing.
+## List of quick starts
 
-The folder structure for the Quickstart is as follows:
+- [Sample Model Deployment](./basic/README.md)
+- [Sample Model Deployment and Autoscaler](./hpa/README.md)
+- [Sample Model Deployment and Persistent Volume Claim](./pvc/README.md) 
```

Comment:
Sample Model Deployment and Persistent Volume Claim would be like this
Sample Model Deployment using Persistent Volume Claim